### DISCUSSION TITLE
Cannot use manual sorting in the card layout

### DISCUSSION THREAD

**Discussion by HitomiTenshi** [directus/directus#8141] [Feature Requests]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

We have a categories table with small preview images of the category. The view is set to card layout by default but when we need to use manual sorting we need to switch back to the table layout. It would be nice if the card layout could also activate manual sorting.

I think it would make sense when manual sorting in the card layout goes from top left (first entry) to bottom right (last entry).

### To Reproduce

1. Have a sort field on your table.
2. Select the card layout.
3. You cannot use manual sorting now.

### What version of Directus are you using?

v9.0.0-rc.93

### What version of Node.js are you using?

14.17.6

### What database are you using?

MariaDB 10.3.31

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### How are you deploying Directus?

running locally via npm package

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
